sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubClient.java

```diff
@@ -506,9 +506,9 @@ public abstract List<SubscriptionPath> listSubscriptions(ProjectPath project, To
    */
   public abstract boolean isEOF();
 
-  /** Create {@link com.google.api.services.pubsub.model.Schema} from resource path. */
+  /** Create {@link com.google.api.services.pubsub.model.Schema} from Schema definition content. */
   public abstract void createSchema(
```

Comment:
I understand this method is just introduced last week (#24443) and should not introduce breaking change to released versions. However I would like to understand how "public" the PubSubClient abstract class is? Suppose users have their own pubsub client implementation adding a new abstract method would break them. Or, users are not supposed to implement their own client but the "public" here is for the usage of other modules within Beam SDK. In the latter case, we should add some javadoc to the class warning that "internal use only, no backward compatibility guaranteed."

Reply:
Unfortunately PubsubClient has been around for several years.

Marking with `@Internal` makes sense and can be done in a follow-up. There are likely several classes in here that should be updated and we could add a note in `CHANGES.md` that we annotated classes with `@Internal`.